### PR TITLE
Added tanisha-gupta details

### PR DESCRIPTION
**Pull request by tanishagupta1**

### CODE REVIEW

File: public/data/tanishagupta1.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Tanisha Gupta",
+  "bio": "Software Developer| Tech-Writer | Open Source Enthusiast",
```
Comment (Panquesito7) [inline public/data/tanishagupta1.json:3]:
```suggestion
  "bio": "Software Developer | Tech-Writer | Open Source Enthusiast",
```

**Review by Panquesito7 — COMMENTED** [review]
Next time, please do not make multiple PRs. Instead, make the necessary changes in the same PR, as it might confuse maintainers. 😄

File: public/data/tanishagupta1.json (hunk 1 above)
Comment (tanishagupta1) [reply public/data/tanishagupta1.json:3]: Yes for Sure, Sorry for the inconvenience.

**Review by ssensalo — APPROVED** [review]

File: public/data/tanishagupta1.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Tanisha Gupta",
+  "bio": "Software Developer | Tech-Writer | Open Source Enthusiast",
+  "avatar": "https://github.com/tanishagupta1.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/tanishagupta1",
```
Comment (Panquesito7) [inline public/data/tanishagupta1.json:8]:
```suggestion
      "url": "https://github.com/tanishagupta1",
```

File: public/data/tanishagupta1.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/tanisha_gupta12",
```
Comment (Panquesito7) [inline public/data/tanishagupta1.json:13]:
```suggestion
      "url": "https://twitter.com/tanisha_gupta12",
```

File: public/data/tanishagupta1.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "http://linkedin.com/in/tanisha-gupta1",
```
Comment (Panquesito7) [inline public/data/tanishagupta1.json:18]:
```suggestion
      "url": "https://www.linkedin.com/in/tanisha-gupta1",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please check the comments by @Panquesito7

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**